### PR TITLE
New topbar

### LINKED ISSUE #251: Please add the SOX button to the new topbar.
### Current Behaviour
The SOX button doesn’t appear in the new topbar:
No SOX Button

### SOX Errors logged to the browser console -- F12 (if available)
N/A

### Steps to reproduce
1. Visit [your profile](http://stackoverflow.com/users/preferences/current#newheader) and check the last checkbox.
2. Wait 6-8 minutes.
3. Visit Stack Overflow.
4. The SOX button is missing.

### PR DESCRIPTION
**Pull request by j-f1**
Makes SOX work on the new topbar.

Fixes #251.

### CODE REVIEW

File: sox.user.js [hunk 1]
```diff
@@ -52,6 +52,7 @@
     'use strict';
 
     function init() {
+        sox.NEW_TOPBAR = !!$('.js-so-header').length
```
Comment (enki-code) [inline sox.user.js]: The topbar initialization code needs to be moved to after the location check.

Comment (j-f1) [inline sox.user.js]: @enki-code Done.

**Comment by shu8** [comment]
@j-f1 thanks for the pull request! I'll try and get this merged ASAP :)

**Comment by j-f1** [comment]
@shu8 Any updates?

**Comment by shu8** [comment]
Sorry -- I've been away for a few days so couldn't get around to this. I'll
get this done by tomorrow hopefully!

Sorry for the time this is taking :(

**Comment by j-f1** [comment]
No problem :)

**Comment by shu8** [comment]
@j-f1 thanks! I finally got round to it, merge in #255! :)

**Pull request closed without merging**